packages/plugins/apq/src/index.ts

```diff
@@ -36,6 +36,9 @@ export interface APQOptions {
     str: string,
     api: { crypto: Crypto; TextEncoder: typeof TextEncoder },
   ) => PromiseOrValue<string>;
+  resultConfig?: {
+    forceStatusCodeOk?: boolean;
```

Comment:
Can we add as small description here, as well as on the documentation?
Should we maybe make this the default and release it as a breaking change?

Reply:
Not sure if it should be the default.
Discussed it with @ardatan, he tested it and it agains Apollo Client and updated it works fine as-is.